.github/workflows/ci.yml

```diff
@@ -27,6 +27,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3.0.0
+      - uses: actions/setup-go@v2
```

Comment:
Does v3 of this action a cause an issue? If not, will you upgrade this to v3?

Reply:
I'll check it 👍